client/tests/integration/sorting.rs

```diff
@@ -55,7 +55,7 @@ fn correct_pagination_assets_after_creating_new_one() {
         asset_metadata
             .insert_with_limits(
                 sort_by_metadata_key.clone(),
-                i as u32,
+                JsonString::new(&i),
```

Comment:
```suggestion
                JsonString::new(&i),
```

same, reference to a `Copy` type